integration_test/third_party_apps_data/applications/vault/debian_ubuntu/install

```diff
@@ -80,7 +80,13 @@ else
   sudo apt-get update && sudo apt-get install -y gpg wget
   wget -O- https://apt.releases.hashicorp.com/gpg | gpg --dearmor | sudo tee /usr/share/keyrings/hashicorp-archive-keyring.gpg >/dev/null
   gpg --no-default-keyring --keyring /usr/share/keyrings/hashicorp-archive-keyring.gpg --fingerprint
-  echo "deb [signed-by=/usr/share/keyrings/hashicorp-archive-keyring.gpg] https://apt.releases.hashicorp.com $(lsb_release -cs) main" | sudo tee /etc/apt/sources.list.d/hashicorp.list
+  source /etc/os-release
+  if [ "${VERSION_CODENAME}" = "lunar" ]; then
+    # currently ubuntu 23.04 (lunar lobster) is not supported, install jammy (ubuntu 22.04) package instead
```

Comment:
Our customers aren't going to do that, are they? Should we just disable vault support on Ubuntu 23.04 instead?